api/openapi-spec/openapi.yaml

```diff
@@ -340,6 +403,21 @@ components:
         userAttributes:
           type: object
           additionalProperties: true
+    DecideContext:
+      properties:
+        decideOptions:
+          type: array
+          items:
+                type: string
+          example:
+            [DISABLE_DECISION_EVENT,ENABLED_FLAGS_ONLY,IGNORE_USER_PROFILE_SERVICE,EXCLUDE_VARIABLES,INCLUDE_REASONS]
+        userId:
```

Comment:
I think you can define all these possible values as enums. so that open spec fails if any other string option is passed within the options